lib/fingerprint.js

```diff
@@ -14,3 +20,28 @@ export default function fingerprint(file, options) {
     options.endpoint
   ].join("-");
 }
+
+function reactNativeFingerprint(file, options) {
+  let exifHash = file.exif ? hashCode(JSON.stringify(file.exif)) : "noexif";
```

Comment:
What does `hashCode` exactly do? What kind of hash does it calculate? Also what does `file.exif` look like?

Reply:
the `file.exif` is an object containing various properties about the file. So we are hashing the entire content for uniqueness.

Reply:
Yes, but what kind of hash? It's obviously not a cryptographic one. Why did you choose to imitate Java's `hashCode`. It would be nice if you shared a bit of your thinking behind those decisions.

Reply:
tbh there was little consideration from my end. The snippet was recommended by @kvz during a pairing session, and I simply rocked with it.

Reply:
@Acconut is there a different approach you would rather recommend here?

Reply:
I don't have a concrete alternative but I prefer to know the intent behind code like this. So if we want to change it in the future we know what to look out for.